lib/src/main/res/layout/list_item_color.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Good call. I would add that as a comment on the #10 ticket. I don't think it's necessary for this PR.